.github/workflows/release.yaml

```diff
@@ -90,10 +91,14 @@ jobs:
           path: homebrew-tap
       - name: Update homebrew
         working-directory: homebrew-tap
+        env:
+          GORELEASER_ARTIFACTS: "${{ steps.goreleaser.outputs.artifacts }}"
         run: |
+          RELEASE_SHA256=$(echo $GORELEASER_ARTIFACTS | jq -r '.[] | select(.type == "Archive") | .extra.Checksum | split(":")[1]')
+          [ -z "$RELEASE_SHA256" ] && exit 1
           cd Formula
-          awk -v ver="${RELEASE_VERSION:1}" '/version/, /tags/ {sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)} {print}' kubectl.rb > tmp.rb
-          mv tmp.rb kubectl.rb
+          awk -v ver="${RELEASE_VERSION:1}" '/kubectl_opslevel_version/ {sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)} {print}' kubectl.rb > tmp.rb
+          awk -v new_sha="${RELEASE_SHA256}" '/sha256/ {sub(/[0-9a-f]{10}+/, new_sha)} {print}' tmp.rb > kubectl.rb
```

Comment:
awk regex breakdown:
`sub(`    // substitute (this line only, once)
`/[0-9a-f]{10}+/`      // Find 10 characters, only numbers or lowercase a-f. Followed by + (all additional number, a-f chars)

Reply:
TIL that sha256 only uses `[0-9a-f]`